examples/MigrateUsersToPro.ps1

```diff
@@ -0,0 +1,184 @@
+﻿<#
+.Synopsis
+    Migrates users to Pro Power BI licenses.
+.Description
+    Migrates standard Power BI licensed users to Power BI Pro licenses.
+    If users witout standard licenses should be migrated, include the -MigrateNonStandardPowerBIUsers switch.
+    
+    This script uses the MSOnline PowerShell module. if this module isn't installed it will be installed into your the current user profile.
+.Parameter Path
+    Path to CSV containing users (PUIDs) to migrate to Power BI Pro licenes.
+.Parameter PUIDColumnName
+    Name of column in CSV that contains the PUID for the user to migrate. Default is 'Puid'.
+.Parameter LicenseTypeScan
+    License type for the standard user or if you want to only migrate users containing a certain license type. Ignore if -MigrateNonStandardPowerBIUsers is specified. Default is 'POWER_BI_STANDARD'.
+.Parameter ProLicenseName
+    Name of Pro Power BI license. Default is 'POWER_BI_PRO'.
+.Parameter MigrateNonStandardPowerBIUsers
+    Indicates to migrate users without Power BI standard licenses. By default only standard Power BI users are migrated to Pro licenses.
+.Parameter NoResultFile
+    Indicates to not write result files for the execution of the script. Result still shows in console output.
+.Example
+    PS C:\> .\MigrateUsersToPro.ps1 -Path .\usersToMigrate.csv
+
+    CSV has a column named 'Puid', IDs in column will be migrated to Pro Power BI licesnes (if possible).
+#>
+[CmdletBinding()]
+param
+(
+    [Parameter(Mandatory)]
+    $Path,
+
+    [ValidateNotNullOrEmpty()]
+    [string] $PUIDColumnName = 'Puid',
+
+    [ValidateNotNullOrEmpty()]
+    [string] $LicenseTypeScan = 'POWER_BI_STANDARD',
```

Comment:
This looks like it's unused.

Reply:
Good catch, I was able to remove a few things thanks to that